examples/README.md

```diff
@@ -12,23 +12,34 @@ _Read this in other languages_: [日本語](README.ja.md)
 - Developer libraries (gcc/python dev headers) installed on machine you intend to run on
 
 **Amazon Linux**
+
 ```bash
-sudo yum groupinstall "Development Tools" 
+sudo yum groupinstall "Development Tools"
 sudo yum install python3-devel
 ```
+
 **Ubuntu**
+
 ```bash
 sudo apt install build-essential
 sudo apt-get install python3-dev
 ```
+
 **OSX**
+
 ```bash
 xcode-select --install
 ```
 
+## To Pattern Match or Not to Pattern Match?
+
+Python introduced [structural pattern matching](https://peps.python.org/pep-0636/) in version 3.10. We believe your experience with our client will really shine with this feature. To demonstrate this, we provide examples with and without pattern matching. The example filenames without pattern matching are suffixed with `_prepy310`. For example, `example_async.py` demonstrates our asynchronous client _with_ pattern matching; `example_async_prepy310.py` demonstrates our asynchronous client _without_ pattern matching.
+
+If you are new to this feature, we invite you to try it. Nevertheless you do _have_ to use it. Our client is perfectly compatible with the legacy `if-elif-else` type checking with `isinstance`.
```

Comment:
This is great but I think it belongs in the main README in addition to or instead of here?  Check out the "Response Types" sections in the JS/.NET READMEs

Reply:
I put the blurb in the main README.